packages/wdio-reporter/reporter.d.ts

```diff
@@ -26,9 +26,9 @@ declare namespace WDIOReporter {
     }
 
     interface Options {
-        configFile: string;
-        logFile: string;
-        logLevel: string;
+        configFile?: string;
```

Comment:
Are you sure these should be optional?

Reply:
Yes. Because there are tests where we are checking if parameter are not passed, then default value should be selected, and if those are not optional then , it is giving compilation issue

Reply:
So, as a user I can omit passing these params and be sure that some default values are used?
If so, ok for me.
However, if we need the change just to test something let's revert

Reply:
> So, as a user I can omit passing these params and be sure that some default values are used?
Yes, as per my understanding, that is what happening in the script at below line:

https://github.com/webdriverio/webdriverio/blob/e016d1071be05ebec6126814f4ffc6ad5fa8beb8/packages/wdio-dot-reporter/src/index.ts#L16 

> If so, ok for me.
> However, if we need the change just to test something let's revert

Reply:
According to that line it looks like they should be required?

Reply:
are you sure about that? because it is mentioned in the comment as well that 

**make dot reporter to write to output stream by default**
        options = Object.assign({ stdout: true }, options)

Reply:
I think I'm missing something, let's keep it as is for now then

Reply:
how can i handle these test, if we revert the changes

https://github.com/suniljaiswal01/webdriverio/blob/typescript-dot-reporter/packages/wdio-dot-reporter/tests/index.test.ts#L25-L28

There are giving compile error

Reply:
@mgrybyk I don't see `configFile` ever been used in any of the reporters. I would even say we should remove this from the interface.

WDYT?